plugins/manipulator.cpp

```diff
@@ -32,9 +32,12 @@
 #include "df/caste_raw.h"
 #include "df/historical_entity.h"
 #include "df/entity_raw.h"
+#include "df/goal_type.h"
+#include "df/unit_personality.h"
 
 #include "uicommon.h"
 #include "listcolumn.h"
+#include "manipulator.h"
```

Comment:
Looks like you either forgot to add this file or this was a holdover from an earlier change:
https://github.com/DFHack/dfhack/pull/1615/checks?check_run_id=946915877#step:10:383

Reply:
Oops, that's from a refactor I backed out of, Visual Studio trying too be too helpful. Will remove...